tests/test_image_processing_common.py

```diff
@@ -241,6 +241,16 @@ def measure_time(image_processor, image):
 
         self.assertLessEqual(fast_time, slow_time)
 
+    def test_is_fast(self):
+        for image_processing_class in self.image_processor_list:
+            image_processor = image_processing_class(**self.image_processor_dict)
+
+            # Check is_fast is set correctly
+            if "Fast" in image_processing_class.__name__:
```

Comment:
Let's use this  instead here
```suggestion
            if isinstance(image_processing_class, BaseImageProcessorFast):
```